examples/sheneos/sheneos_compare.cpp

```diff
@@ -24,7 +24,7 @@
 #include <vector>
 
 #include "sheneos/interpolator.hpp"
-#include "fname.h"
+#include "fname.hpp"
```

Comment:
This shouldn't be needed anymore as it defines interface-compatibility of external symbols with fortran.

Reply:
Good catch. This whole file is actually also unnecessary as it only serves as a shim to the fortran functionality. I've removed both.